.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.7.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+  webhint:
+    name: Webhint
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Webhint
+        run: |
+          npm install --save-dev hint@7.x
+          [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.hintrc
+      - name: Webhint Report
+        run: npx hint .
+  stylelint:
+    name: Stylelint
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
+  nodechecker:
+    name: node_modules checker
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check node_modules existence
+        run: |
+          if [ -d "node_modules/" ]; then echo -e "\e[1;31mThe node_modules/ folder was pushed to the repo. Please remove it from the GitHub repository and try again."; echo -e "\e[1;32mYou can set up a .gitignore file with this folder included on it to prevent this from happening in the future." && exit 1; fi
```

Comment:
- [ ] Great job on adding the linters file but you need also to add the files for `hintrc` to check for HTML linters and `stylelint` to check for CSS linters so kindly add them to your project 

![image](https://user-images.githubusercontent.com/58553711/197897397-4ab83e38-1728-4992-924e-f9e26e1080ec.png)

- [ ] I think you didn't install the packages for hintrc and stylelint so kindly if you scroll down in this [Linters for HTML and CSS](https://github.com/microverseinc/linters-config/tree/master/html-css)you will see some steps on how to install them then commands to check if there's any linters error 🔥🚀💯

![image](https://user-images.githubusercontent.com/58553711/197914603-942a453e-066b-49c9-9ecd-8bbf4f869f0d.png)


## IF you scroll dow you will see the commands so you can check if there's any linters error need to fix 💯☺💪🏻
- This Command to check for HTML linters errors
![image](https://user-images.githubusercontent.com/58553711/197897533-9fca57f9-2f2b-4e38-ba1f-eb914601918c.png)

- This Command to check for CSS linters errors 💯
![image](https://user-images.githubusercontent.com/58553711/197897653-3182476c-2a50-48b4-bab7-3cebd112c3d1.png)

Reply:
Hello Rem-Lab

I resolved the required changes, my question is: Do I need to ask for a new review?, Sorry, I'm still learning...